src/backend/base/langflow/components/data/google_oauth_token.py

```diff
@@ -83,8 +83,7 @@ def build_output(self) -> Data:
                 flow = InstalledAppFlow.from_client_secrets_file(client_secret_file, scopes)
                 creds = flow.run_local_server(port=0)
 
-            with token_path.open(mode="w", encoding="utf-8") as token_file:
-                token_file.write(creds.to_json())
+                token_path.write_text(creds.to_json(), encoding="utf-8")
```

Comment:
Is the alignment correct ? It has shifted from previous code...

Reply:
Yes, that's correct, the functions were working normally but, thanks to your help with Path.write_text(...) I realized that the token was being written every time it was executed even when there was no need for changes, and so I put it inside the if so that it only executes when the requirements are met.

Reply:
👍